Doc/whatsnew/3.11.rst

```diff
@@ -1173,6 +1173,61 @@ Deprecated
   (Contributed by Serhiy Storchaka in :gh:`91760`.)
 
 
+Pending Removal in Python 3.12
+==============================
+
+The following APIs have been deprecated in earlier Python releases, and are now
+pending removal in Python 3.12.
```

Comment:
This could be clearer.  I'm not sure if it's telling me that they will be removed in 3.12, or if they are marked as "pending removal" (i.e. deprecated?) in 3.12.

Reply:
Perhaps:

```restructuredtext
The following APIs have been deprecated in earlier Python releases, 
and will be removed in Python 3.12.
```

There's the possibility that removal will be delayed, but I think the stronger language is OK as it covers the vast majority of languages (if it isn't, we could go back to "are scheduled for removal in ...")

A

Reply:
Agreed with @vstinner ; its much better that users assume they will be removed and update their code accordingly and then the removal be deferred a version than the inverse.